Gems/ScriptCanvasTesting/Code/Source/Framework/ScriptCanvasTestFixture.h

```diff
@@ -193,6 +194,17 @@ namespace ScriptCanvasTests
             return m_graph;
         }
 
+        ScriptCanvasEditor::EditorGraph* CreateEditorGraph()
+        {
+            if (m_graph == nullptr)
+            {
+                m_graph = aznew ScriptCanvasEditor::EditorGraph();
+                m_graph->Init();
+            }
+
+            return static_cast<ScriptCanvasEditor::EditorGraph*>(m_graph);
+        }
+
```

Comment:
This doesn't look safe. The `CreateGraph` method above will create an instance of `ScriptCanvas::Graph` if `m_graph` does not exist. But this function will similarly create an instance of `ScriptCanvasEditor::EditorGraph` if it doesn't exist, so what you get will depend on who gets called first. Should the editor graph be stored in a different variable?

Reply:
Each test that uses CreateGraph() or CreateEditorGraph() should really only use one or the other once to determine the type of graph the test is working with. And I don't see much point in converting between them in the middle of a test, since an EditorGraph is a Graph. I probably shouldn't create a separate variable, because the other functions in ScriptCanvasTestFixture use m_graph specifically